io/eolearn/io/processing_api.py

```diff
@@ -0,0 +1,205 @@
+''' An input task for the `sentinelhub processing api <https://docs.sentinel-hub.com/api/latest/reference/>`
+'''
+import json
+import logging
+from copy import deepcopy
+import datetime as dt
+import numpy as np
+
+from sentinelhub import WebFeatureService, MimeType, SentinelHubDownloadClient, DownloadRequest, SHConfig
+import sentinelhub.sentinelhub_request as shr
+
+from eolearn.core import EOPatch, EOTask, FeatureParser
```

Comment:
Instead of importing `FeatureParser` you can use `self._parse_features` method inside a task, which does the same.